### PR TITLE
fix: [#6825]Update old packages

### PR DESCRIPTION
**Pull request by JhontSouth**
Addresses # 6825
#minor

## Description
This PR upgrades old packages that don't have conflicts with BotBuilder in their latest versions.

## Specific Changes
  - Updated the following packages:
      - AdaptiveCards 
      - Azure.Storage.Blobs 
      - Azure.Storage.Queues 
      - BenchmarkDotNet 
      - coverlet.collector 
      - coverlet.msbuild
      - JsonPath.Net 
      - Microsoft.ApplicationInsights 
      - Microsoft.ApplicationInsights.AspNetCore 
      - Microsoft.ApplicationInsights.DependencyCollector 
      - Microsoft.ApplicationInsights.PerfCounterCollector 
      - Microsoft.ApplicationInsights.TraceListener 
      - Microsoft.ApplicationInsights.Web 
      - Microsoft.ApplicationInsights.WindowsServer 
      - Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel 
      - Microsoft.AspNet.WebApi 
      - Microsoft.AspNetCore.Hosting.Abstractions 
      - Microsoft.AspNetCore.Http 
      - Microsoft.AspNetCore.Mvc.Core 
      - Microsoft.AspNetCore.Mvc.Formatters.Json 
      - Microsoft.AspNetCore.Mvc.NewtonsoftJson 
      - Microsoft.AspNetCore.Mvc.Testing
      - Microsoft.Azure.Cosmos
      - Microsoft.CodeAnalysis.CSharp.Scripting
      - Microsoft.CSharp 
      - Microsoft.Extensions.Configuration 
      - Microsoft.Extensions.Configuration.Json 
    - Microsoft.Extensions.Configuration.UserSecrets 
    - Microsoft.Extensions.DependencyInjection
    - Microsoft.IdentityModel.Protocols.OpenIdConnect 
    - Microsoft.IdentityModel.Tokens
    - Microsoft.NET.Test.Sdk
    - MSTest.TestAdapter 
    - MSTest.TestFramework 
    - NAudio 
    - Newtonsoft.Json
    - NuGet.Packaging 
    - NunitXml.TestLogger 
    - ReportGenerator
    - SlackAPI 
    - System.Drawing.Common 
    - System.IdentityModel.Tokens.Jwt 
    - System.IO.Pipelines 
    - System.Net.Security 
    - System.Text.Encodings.Web 
    - xunit.runner.visualstudio 
    - XunitXml.TestLogger  

## Testing
The following image shows the unit tests working after the updates.
image

### CODE REVIEW

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Slack.Tests/Microsoft - Backup.Bot.Builder.Adapters.Slack.Tests.csproj
Comment (sw-joelmut) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Slack.Tests/Microsoft - Backup.Bot.Builder.Adapters.Slack.Tests.csproj:1]: I think this file doesnt go right?, it says Backup...

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/Microsoft - Backup.Bot.Builder.Adapters.Twilio.Tests.csproj
Comment (sw-joelmut) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/Microsoft - Backup.Bot.Builder.Adapters.Twilio.Tests.csproj:1]: Same for this file

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.Tests/Microsoft - Backup.Bot.Builder.Adapters.Webex.Tests.csproj
Comment (sw-joelmut) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.Tests/Microsoft - Backup.Bot.Builder.Adapters.Webex.Tests.csproj:1]: Same here

File: tests/Microsoft.Bot.AdaptiveExpressions.Core.AOT.Tests/Microsoft - Backup.Bot.AdaptiveExpressions.Core.AOT.Tests.csproj
Comment (sw-joelmut) [inline tests/Microsoft.Bot.AdaptiveExpressions.Core.AOT.Tests/Microsoft - Backup.Bot.AdaptiveExpressions.Core.AOT.Tests.csproj:1]: same here

File: tests/Microsoft.Bot.AdaptiveExpressions.Core.Tests/Microsoft - Backup.Bot.AdaptiveExpressions.Core.Tests.csproj
Comment (sw-joelmut) [inline tests/Microsoft.Bot.AdaptiveExpressions.Core.Tests/Microsoft - Backup.Bot.AdaptiveExpressions.Core.Tests.csproj:1]: same here

File: tests/Microsoft.Bot.Builder.AI.LUIS.Tests/Microsoft - Backup.Bot.Builder.AI.Luis.Tests.csproj
Comment (sw-joelmut) [inline tests/Microsoft.Bot.Builder.AI.LUIS.Tests/Microsoft - Backup.Bot.Builder.AI.Luis.Tests.csproj:1]: same here

File: tests/Microsoft.Bot.Builder.AI.Orchestrator.Tests/Microsoft - Backup.Bot.Builder.AI.Orchestrator.Tests.csproj
Comment (sw-joelmut) [inline tests/Microsoft.Bot.Builder.AI.Orchestrator.Tests/Microsoft - Backup.Bot.Builder.AI.Orchestrator.Tests.csproj:1]: same here, check all Backup. files

**Review by sw-joelmut — CHANGES_REQUESTED** [review]

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Slack.Tests/Microsoft - Backup.Bot.Builder.Adapters.Slack.Tests.csproj
Comment (JhontSouth) [reply tests/Adapters/Microsoft.Bot.Builder.Adapters.Slack.Tests/Microsoft - Backup.Bot.Builder.Adapters.Slack.Tests.csproj:1]: I removed all the Backup files

File: tests/Microsoft.Bot.Builder.AI.Orchestrator.Tests/Microsoft - Backup.Bot.Builder.AI.Orchestrator.Tests.csproj
Comment (JhontSouth) [reply tests/Microsoft.Bot.Builder.AI.Orchestrator.Tests/Microsoft - Backup.Bot.Builder.AI.Orchestrator.Tests.csproj:1]: I removed all the Backup files

**Review by sw-joelmut — APPROVED** [review]

**Comment by JhontSouth** [comment]
Promoted /6830

**Pull request closed without merging**